template/Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
-ruby File.read(File.join(__dir__, '.ruby-version')).strip
+ruby '>= 2.6.10'
```

Comment:
Let's pick a particular version here, because we need to pick one for the .ruby-version file anyway. I'd use 2.7.6, because Facebook people recommend it in the GH issues sometimes.

Reply:
That's how done in 0.71.6 RN default template, they deleted `.ruby-version`, and put this version definition here, but yep, technically make sense to bring  `.ruby-version` back and put 2.7.6, I like this version too